spec/ruby/optional/capi/ext/object_spec.c

```diff
@@ -410,6 +410,16 @@ static VALUE object_spec_custom_alloc_func_p(VALUE self, VALUE klass) {
   return allocator ? Qtrue : Qfalse;
 }
 
+static VALUE object_spec_redefine_rb_obj_frozen_p(VALUE self) {
```

Comment:
```suggestion
static VALUE object_spec_redefine_frozen(VALUE self) {
```
`redefine_rb_obj_frozen_p` sounds too confusing to me, we can't redefine a C function